portfolio/src/main/webapp/gameoflife.js

```diff
@@ -16,7 +16,7 @@ resizeCanvasToDisplaySize(canvas);
 // Game of Life specific variables
 const SPEED = 200; // in miliseconds (speed between updates)
 const CELL_SIZE = 10; // in pixels (side length of individual cell)
-const SEED_SPAWN_RATE = 0.1; // as a ratio (likelihood a cell is 'alive' when seeded)
+const SEED_SPAWN_RATE = 0.05; // as a ratio (likelihood a cell is 'alive' when seeded)
```

Comment:
Just curious why you made this change?

Reply:
Before the change, the board would converge to an "end state" (where most of the board is covered) within 10 seconds, but decreasing it greatly slows the process which makes it (imo) a little nicer to look at.